tests/core_tests/tx_validation.cpp

```diff
@@ -212,7 +212,7 @@ bool gen_tx_unlock_time::generate(std::vector<test_event_entry>& events) const
 
   GENERATE_ACCOUNT(miner_account);
   MAKE_GENESIS_BLOCK(events, blk_0, miner_account, ts_start);
-  REWIND_BLOCKS_N(events, blk_1, blk_0, miner_account, 10);
+  REWIND_BLOCKS_N(events, blk_1, blk_0, miner_account, 20);
```

Comment:
Because I attempt to make more transactions (they fail) and the way the tx construction works is that I need extra outputs in my account even if the tx construction fails.